queries/scala/function_definition.scm

```diff
@@ -0,0 +1,5 @@
+(function_declaration
+      name: (identifier) @method)
+
+(function_definition
+      name: (identifier) @method)
```

Comment:
```suggestion
(function_definition) @fn
(lambda_expression) @fn
```

Again, removed `function_declaration` and added `lambda_expression`.
Also, you were only capturing the function's name, but it needed the entire function node